styleguide/source/_patterns/02-organisms/06-topic/06-topic-promotion/topic-promotion.twig

```diff
@@ -1,10 +1,20 @@
-<div class="topic_promotion {{ class }}">
+<div class="topic_promotion {{ class }} {% if promo.border %}topic_promotion-border{% endif %}">
     {% set tag = promo.cta ? 'div': 'a' %}
     <{{tag}} class="topic_promotion_container" href="#">
-        {% include 'atoms-h2' with {'content': promo.title, 'class': default } %}
-        {% include 'atoms-paragraph' with {'content': promo.content, 'class': default } %}
-        {% if block.button %}
-            {% include 'atoms-button-link' with {'class': 'block_cta-with-list-button-link' } %}
+    {% if promo.logo %}
+            {% include 'atoms-logo' with { 'link': '#', 'image': promo.logo, 'alt': 'logo', 'class': default } %}
         {% endif %}
+        <div class="topic_promotion_image">
```

Comment:
Suggestion to include <div class="topic_promotion_image"> in the {% if promo.image %} if statement so that an unused <div> does not print when there is not an image.